site/src/layout/header/MobileMenu.tsx

```diff
@@ -0,0 +1,284 @@
+                                                                    {` | ${node.name}`}
+                                                                </Typography>
+                                                            </OverviewButton>
+                                                        </li>
+                                                        {node.childNodes.map((node) => (
+                                                            <li key={node.id}>
+                                                                <Link page={node} aria-label={node.name}>
+                                                                    {node.name}
+                                                                </Link>
+                                                            </li>
+                                                        ))}
+                                                    </PageLayoutContent>
+                                                </PageLayout>
+                                            </SubLevelNavigation>
+                                        )}
+                                    </li>
+                                ))}
+                            </TopLevelNavigation>
+                        </nav>
+                    </PageLayoutContent>
+                </PageLayout>
+            </MenuContainer>
+        </Root>
+    );
+};
+
+const Root = styled.div`
+    ${({ theme }) => css`
+        ${theme.breakpoints.sm.mediaQuery} {
+            display: none;
+        }
+    `}
+`;
+
+const PageLayoutContent = styled.div`
+    grid-column: 2 / -2;
+`;
+
+const MenuButton = styled.button`
+    appearance: none;
+    border: none;
+    background-color: transparent;
+    color: inherit;
+    padding: 0;
+    width: 24px;
+    height: 24px;
+`;
+
+const IconWrapper = styled.div`
+    width: 16px;
+    height: 16px;
+`;
+
+const Icon = styled(SvgUse)`
+    width: 100%;
+    height: 100%;
+    color: inherit;
+`;
+
+const MenuContainer = styled.div<{ $isMenuOpen: boolean }>`
+    display: block;
+    position: fixed;
+    top: var(--header-height);
+    left: 0;
+    height: 0;
+    width: 100vw;
+    z-index: 40;
+    background-color: ${({ theme }) => theme.palette.gray["200"]};
+    overflow: auto;
+    visibility: hidden;
+    transition: height 0.15s ease-out, visibility 0s linear 0.15s;
+
+    ${({ $isMenuOpen }) =>
+        $isMenuOpen &&
+        css`
+            visibility: visible;
+            height: calc(100vh - var(--header-height));
+            transition: height 0.25s ease-in;
+        `}
+`;
+
+const TopLevelNavigation = styled.ol`
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+`;
+
+const SubLevelNavigation = styled.ol<{ $isExpanded: boolean }>`
+    list-style-type: none;
+    position: fixed;
+    top: var(--header-height);
+    left: 100vw;
+    height: calc(100vh - var(--header-height));
+    width: 100vw;
+    z-index: 41;
```

Comment:
Since this is already nested inside `MenuContainer`, wouldn't it be enough to just span the full width and height? 🤔 

```suggestion
    position: absolute;
    inset: 0;
    z-index: 1; // Maybe not even necessary
```

Reply:
🤔  you missed the transition / animation from the right side, that does not work with position: absolute;

Reply:
You can use `transformX` for the animation, that's probably smoother than `left` anyway

Reply:
in the chrome device dev tools the screen is extended (scrollable to right) when position absolute is used

Reply:
You probably need to set `overflow: hidden` on some parent then... but if it works fine with `fixed` then I'm fine with that too 🙂

Reply:
overflow is set to auto on the parent due to https://github.com/vivid-planet/comet-starter/pull/347#discussion_r1760641149